ARTICLE-7.markdown

```diff
@@ -39,7 +39,7 @@ disbursement to Members.
     No less than 20% of the Member Account distributions shall be
     paid out within 8.5 months of the of the fiscal year by cash
     or check. The remaining 80% shall be paid to the member within
-    3 years. Each member of the co-op shall receive a qualified
+    3 years. Each member of the Cooperative shall receive a qualified
     written notice of allocation for the noncash portion of the
     Member Account distribution.
 
```

Comment:
This is separate from charitable donations, if that's what you are referring to. This section concerns Member vs Collective Accounts. Default is 50/50, but subsection e states the membership can vote to adjust if necessary.